### DISCUSSION TITLE
🥂 iLogtail开源社区线上Meetup第3期 11.2 19:30，内有惊喜，欢迎点击！

### DISCUSSION THREAD

**Discussion by yyuuttaaoo** [alibaba/ilogtail#489] [Announcements]
# 概要
* 议题：iLogtail开源社区线上Meetup
* 时间：2022年11月2日19:30
* 直播地点： [B站直播间](https://live.bilibili.com/25786263)
* 参与人：[@messixukejia](https://github.com/messixukejia)、[@yyuuttaaoo](https://github.com/yyuuttaaoo)、[@henryzhx8 ](https://github.com/henryzhx8)、[@Takuka0311](https://github.com/Takuka0311)
* 回访视频 https://www.bilibili.com/video/BV1CD4y147ma/?spm_id_from=333.999.0.0

# 议题
* iLogtail社区版1.2.1 新功能概览。[@yyuuttaaoo](https://github.com/yyuuttaaoo)
* 短Job日志无损采集Demo。[@yyuuttaaoo](https://github.com/yyuuttaaoo)
* 使用Grok插件解析多种日志格式。[@Takuka0311](https://github.com/Takuka0311)
* 启用插件后如何在SLS中浏览日志上下文。[@henryzhx8 ](https://github.com/henryzhx8)

  <img src="https://user-images.githubusercontent.com/1827594/198574964-d441459b-d1c0-4c71-8790-6f46da43dc72.png" height="50%" width="50%" >

# 留言“为iLogtail打Call”，抽取大会门票
* GOPS全球运维大会2022将于10月28-29日上海召开。阿里云SLS团队也将带来《K8s下可观测数据采集的挑战与技术实践》的主题分享。
  同时为了回馈iLogtail社区贡献者，我们免费送出3张GOPS大会门票，欢迎下方留言“为iLogtail打Call”，我们将抽取3位朋友免费赠与门票。会议链接：https://www.bagevent.com/event/8239219
  <img src="https://user-images.githubusercontent.com/1827594/197974396-8eb273b7-19d0-439f-a6d9-ed2c3afdfb99.png" height="50%" width="50%" >

* COSCon中国开源年会将于10月29-30日召开，活动为线上直播，阿里云SLS团队也将带来《云原生可观测采集器iLogtail架构与应用实践》的主题分享。
  会议链接：https://www.bagevent.com/event/8199059
    <img src="https://user-images.githubusercontent.com/1827594/198575386-00ca6015-9898-42c5-acd8-e01d378ee911.png" height="50%" width="50%" >

**Comment by messixukejia** [comment]
为iLogtail打Call。欢迎参与共建。

**Comment by RoboticShark** [comment]
为iLogtail打Call

**Comment by yuanqj** [comment]
iLogtail顶顶顶！

**Comment by yyuuttaaoo** [comment]
回访视频 https://www.bilibili.com/video/BV1CD4y147ma/?spm_id_from=333.999.0.0
